index.html

```diff
@@ -4240,47 +4240,54 @@ <h2>HTTP</h2>
         </section>
 
         <section class="informative">
-          <h2>JSON Processing</h2>
-
+          <h2>JSON-LD Processing</h2>
           <p>
 While the media types describing conforming documents defined in this
-specification always express JSON-LD, JSON-LD processing is not required to be
-performed, since JSON-LD is JSON. Some scenarios where processing a
-<a>verifiable credential</a> or a <a>verifiable presentation</a> as JSON is
+specification always express JSON-LD, RDF processing is not required to be
+performed, since JSON-LD is a concrete RDF syntax as described in [RDF11-CONCEPTS]. 
+Hence, a JSON-LD document is both an RDF document and a JSON document and correspondingly represents an instance of an RDF data model.
+See <a href="JSON-LD11##relationship-to-rdf">Relationship to RDF</a> for more details.
+          </p>
+          <p>
+Some scenarios where processing a <a>verifiable credential</a> or a <a>verifiable presentation</a> as RDF is
 desirable include, but are not limited to:
           </p>
-
           <ul>
             <li>
-Before securing or after verifying content
-that requires <a href="https://csrc.nist.gov/glossary/term/data_integrity">data
-integrity</a>, such as a
-<a>verifiable credential</a> or <a>verifiable presentation</a>.
+Before securing or after verifying <a>verifiable credentials</a> or <a>verifiable presentations</a>,
+in cases where the issuer has leveraged the <code>@context</code> to provide details regarding 
+their intended expression of the <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>.
```

Comment:
```suggestion
in cases where the issuer made use of the underlying RF Dataset model to provide details regarding 
their intended expression of the <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>.
```

At least that is how I understood what I _think_ you meant...

Reply:
Yes, thank you, much better